string GIVE="handover_give"	# used for handing over an object based on haptic feedback
string RECEIVE="handover_receive	# used when robot listens for haptics feedback and grabs object from human

string type
float32 timeout_seconds


---
# Result
bool success
string error_msg
---
# Feedback
bool status
